kernel/configs/fragments/common/netfilter.conf

```diff
@@ -164,10 +170,11 @@ CONFIG_NF_TPROXY_IPV4=y
 CONFIG_NF_DUP_IPV4=y
 CONFIG_NF_LOG_IPV4=y
 CONFIG_NF_REJECT_IPV4=y
+
+# NF_NAT_IPV4 is removed in future kernel, and is part of whitelist.conf:
 CONFIG_NF_NAT_IPV4=y
-CONFIG_NF_NAT_MASQUERADE_IPV4=y
```

Comment:
Why is this dropped?

Reply:
I mentioned details in the commit msg. NF_NAT_MASQUERADE_IPV4: this is a select, not a tristate, so no value in specifying here. Also, in the future much of the ipv4/ipv6 nat code is combined, so this config will not exist in newer kernels.